config/templates/cloud-config-worker

```diff
@@ -466,8 +465,6 @@ write_files:
            done;
            echo done.'
 
-      sudo rkt rm --uuid-file=/var/run/coreos/decrypt-tls-assets.uuid
```

Comment:
no particular reason, feels more right that way:
 - there were errors reported regarding `rkt rm` erroring
 - rkt-gc is already there,let it do its job